integration/tests/chat/channel-settings.spec.ts

```diff
@@ -25,13 +33,30 @@ test('changing channel banner', async ({ page }) => {
   let channelBannerUrl = await chatPage.getChannelBannerUrl()
   expect(channelBannerUrl).toBeNull()
 
-  await chatPage.setChannelBanner(path.join(__dirname, 'channel-banner.png'))
+  await chatPage.setChannelBanner(TEST_IMAGE_PATH)
   await chatPage.clickChannelSettingsSaveButton()
 
   await chatPage.openChannelSettings()
 
   channelBannerUrl = await chatPage.getChannelBannerUrl()
-  expect(channelBannerUrl).toBeDefined()
+  expect(channelBannerUrl).toContain('/files/channel-images/')
+})
+
+test('changing inappropriate channel banner', async ({ page }) => {
+  if (!CHANNEL_BANNERS) {
+    expect(true).toBe(true)
+    return
+  }
+  await loginPage.navigateTo()
+  await loginPage.loginWith('admin', 'admin1234')
+
+  await chatPage.openChannelSettings()
+
+  await chatPage.setChannelBanner(path.join(TEST_IMAGE_PATH_INAPPROPRIATE))
+  await chatPage.clickChannelSettingsSaveButton()
+
+  const errorMessage = await chatPage.getChannelSettingsSnackbarError()
```

Comment:
Wouldn't it make a lot more sense to show this error within the dialog? Using a snackbar for this seems very weird